docs/topics/pages.md

```diff
@@ -142,6 +142,7 @@ These are used for structuring fields in the interface.
 
 #### Customizing the page editor interface
 
+The page editor can be customized further. See [Customizing the editing interface](/advanced_topics/customization/page_editing_interface).
 The page editor can be customized further. See [Customizing the editing interface](/advanced_topics/customisation/page_editing_interface).
```

Comment:
```suggestion
```

Looks like you added a new line here but did not remove the original.

This is probably why the CI is failing.

Reply:
Thank You! @lb-, this was very helpful and all tests have finally passed.

Reply:
Awesome. Let us know when it's ready for another review.